File: RECRUSION/11/01/2022/13/1/rotate.py
def leftRotate(arr, d, n):
    for i in range(d):
        leftRotatebyOne(arr, n)

# Function to left Rotate arr[] of size n by 1*/ 
def leftRotatebyOne(arr, n):
    temp = arr[0]
    for i in range(n-1):
        arr[i] = arr[i + 1]
    arr[n-1] = temp

arr=[ 1,2,3,4 ]; n=4; k=3
leftRotate(arr,k,n)
print(arr)

File: RECRUSION/11/01/2022/13/1/rotate.js
function rot(arr,n,k){
    for(var i=0;i<k;i++){
        leftrot(arr,n)
    }
}
function leftrot(arr,n){
    temp=arr[0]
    for(var i =0;i<n-1;i++){
        arr[i]=arr[i+1]
        console.log(arr)
    }
    arr[n-1]=temp
}
arr=[ 1,2,3,4 ]; n=4; k=3
rot(arr,n,k)
console.log(arr)



